Katello 4.12 Release Notes (#2836)
theforeman/foreman-documentation: guides/doc-Release_Notes/topics/katello.adoc

[id="katello-headline-features"]
== Headline Features

> === Content Access Modes Removed in Favor of Universal Simple Content Access

Reviewer: ```suggestion
=== Entitlement Mode Removed in Favor of Simple Content Access
```

Simple Content Access is now the only content access mode supported. When upgrading Katello, all existing organizations that use entitlement mode will migrate automatically to Simple Content Access. This change is not reversible. 

Toggling SCA from the Organization Edit page, the Organization Create page, and the Organization List page is no longer available.

=== Removed System Purpose-Related Host Statuses
The following host statuses are removed:
* Subscription Status (this column has also been removed from the All Hosts table)
* System Purpose Status
* SLA Status
* Role Status
* Usage Status
* Addon Status

=== Debian Content Support Expanded
Import and export are now supported for repositories containing debian-type content. Aditionally, the host details page will now display more details for debian packages.

=== Miscellaneous
* Dropped dependencies on 'downshift', 'uglifier', 'minitest-tags', and 'simplecov'.
* Katello has been updated to utilize Ruby 3.
* Various UI tweaks, changes, and bugfixes.

[id="katello-upgrade-warnings"]
== Upgrade Warnings

> Simple Content Access is now the only supported content access mode. Existing organizations will be automatically migrated on Katello upgrade (see more info above). Please note that the 'Subscription - Entitlements' report template does not work with SCA organizations, and will never have any rows. It will be removed in a future release or release candidate. Instead, use the 'Host - Installed Products' report. Additionally, please note that SCA-related API and Hammer endpoints and parameters have not yet been removed, so you will experience errors if you try to use them. They will be removed in a future release or release candidate.

Author: I have separated the paragraphs into individual sentences, should be fixed. Thanks for the heads up.

> Simple Content Access is now the only supported content access mode. Existing organizations may be automatically migrated on Katello upgrade (see more info above). Please note that the 'Subscription - Entitlements' report template does not work with SCA organizations, and will never have any rows. It will be removed in a future release or release candidate. Instead, use the 'Host - Installed Products' report. Additionally, please note that SCA-related API and Hammer endpoints and parameters have not yet been removed, so you will experience errors if you try to use them. They will be removed in a future release or release candidate.

Reviewer: ```suggestion
Simple Content Access is now the only supported content access mode. Existing organizations will be automatically migrated on Katello upgrade (see more info above). Please note that the 'Subscription - Entitlements' report template does not work with SCA organizations, and will never have any rows. It will be removed in a future release or release candidate. Instead, use the 'Host - Installed Products' report. Additionally, please note that SCA-related API and Hammer endpoints and parameters have not yet been removed, so you will experience errors if you try to use them. They will be removed in a future release or release candidate.
```

[id="katello-headline-features"]
== Headline Features

=== Entitlement Mode Removed in Favor of Simple Content Access

> Simple Content Access is now the only content access mode supported.

Reviewer 2: ```suggestion
Simple Content Access (SCA) is now the only content access mode supported.
```

When upgrading Katello, all existing organizations that use entitlement mode will migrate automatically to SCA.
This change is not reversible.

Toggling SCA from the Organization Edit page, the Organization Create page, and the Organization List page is no longer available.

=== Removed System Purpose-Related Host Statuses
The following host statuses are removed:
* Subscription Status (this column has also been removed from the All Hosts table)
* System Purpose Status
* SLA Status
* Role Status
* Usage Status

> * Addon Status

Reviewer 2: Was it spelled like this in the UI? Not spelled Add-on or Add on?

Reviewer: Unfortunately yes :confounded: 

![image](https://github.com/theforeman/foreman-documentation/assets/22042343/0230ce44-83fd-47f7-8b7f-df6020bad3af)

Author: Addons status sounds icky to me. Just like fathers in law. I think it's ok as is?

Reviewer: https://www.theonion.com/william-safire-orders-two-whoppers-junior-1819565735

=== Debian Content Support Expanded
Import and export are now supported for repositories containing debian-type content.

> Aditionally, the host details page will now display more details for debian packages.

Reviewer 2: ```suggestion
Additionally, the host details page will now display more details for debian packages.
```

[id="katello-upgrade-warnings"]
== Upgrade Warnings

> Simple Content Access is now the only supported content access mode.

Reviewer 2: ```suggestion
Simple Content Access (SCA) is now the only supported content access mode.
```

Existing organizations will be automatically migrated on Katello upgrade (see more info above).

> Please note that the 'Subscription - Entitlements' report template does not work with SCA organizations, and will never have any rows.

Reviewer 2: ```suggestion
Please note that the 'Subscription - Entitlements' report template does not work with SCA organizations and will never have any rows.
```

It will be removed in a future release or release candidate.
Instead, use the 'Host - Installed Products' report.

> Additionally, please note that SCA-related API and Hammer endpoints and parameters have not yet been removed, so you will experience errors if you try to use them.

Reviewer 2: ```suggestion
Additionally, please note that SCA-related API and Hammer endpoints and parameters have not yet been removed so you will experience errors if you try to use them.
```

They will be removed in a future release or release candidate.

Certain host statuses have been removed (see more info above).
Please note that system purpose _attributes_ are not being removed, only system purpose-related _host statuses_.
System purpose _attributes_ (role, usage, release version, etc.) are still important for repository enablement and reporting to the Subscriptions service of the Red Hat Hybrid Cloud Console.

[id="katello-deprecations"]
== Deprecations

Deprecated 'Subscription - Entitlements' report template as it does not work with SCA organizations.

> 

Reviewer 2: ```suggestion
```

Reviewer 3: I think we need the extra space there to logically separate the subs entitlement report template deprecation from the the next one:

![image](https://github.com/theforeman/foreman-documentation/assets/14796566/7359263e-a393-43da-97b0-3698837d4f75)

> Deprecated certain SCA-related API and Hammer endpoints and parameters

Reviewer 2: ```suggestion
Deprecated certain SCA-related API and Hammer endpoints and parameters.
```
